scripts/smooth_cal_run.py

```diff
@@ -25,7 +25,7 @@
                              chan_blacklists=a.chan_blacklists, pick_refant=a.pick_refant, freq_threshold=a.freq_threshold,
                              time_threshold=a.time_threshold, ant_threshold=a.ant_threshold, verbose=a.verbose)
     cs.time_freq_2D_filter(freq_scale=a.freq_scale, time_scale=a.time_scale, tol=a.tol,
-                           filter_mode=a.filter_mode, window=a.window, maxiter=a.maxiter, **win_kwargs)
+                           filter_mode=a.filter_mode, window=a.window, maxiter=a.maxiter, method=a.method, **win_kwargs)
```

Comment:
You are missing eigenval_cutoff and skip_flagged_edges

Reply:
Added these